### ISSUE TITLE
Can we implement this code in Android ?

### ISSUE THREAD

**Issue by ubrainrepo** [vasusen-code/VIDEOconvertor#59]
It is possible to implement this code in Android. if yes then how can we implement this code in Android project

**Comment by vasusen-code** [comment]
yeah you can use the ffmpeg part, also you can make better settings according to your software.
